### ISSUE TITLE
iTunes XML dump

### ISSUE THREAD

**Issue by anatosun** [anatosun/plex2mix#1]
Provide a way to dump the library into an iTunes XML format for ease of use with several DJ softwares.

**Comment by cedvan** [comment]
Hi,

Thank you to this tool really usefull !
Personnaly I use rekordbox, so not import rating from m3u8 :(.

Add itunes XML functionnality will be great to include rating sync support and automation !!

**Comment by anatosun** [comment]
I agree with you, `m3u8` is limited in terms of information that you can read from Rekordbox (and others). I'm therefore very much inclined to implement it. However, I have my hands full these days, so I'll have to postpone the implementation.

**Comment by cedvan** [comment]
https://github.com/anatosun/plex2mix/pull/8

**Comment by cedvan** [comment]
PR is ready to merge ;)

**Comment by cedvan** [comment]
Maybe close this issue :) ?
